Documentation/metrics.md

```diff
@@ -14,6 +14,18 @@ The metrics under the `etcd` prefix are for monitoring and alerting. They are st
 
 Metrics that are etcd2 related are documented [here][v2-http-metrics].
 
+### server
+
+These metrics describe the status of the etcd server. In order to detect outages or problems for troubleshooting, the server metrics of every production etcd cluster should be closely monitored.
+
+All these metrics are prefixed with `etcd_server_`
+
+| Name           | Description                       | Type    |
+|----------------|-----------------------------------|---------|
+| leader_changes | The number of leader changes seen | Counter |
```

Comment:
By convention, counters end with _total so this should be `leader_changes_total`.

From a user perspective might it make sense to put "seen" in the name? It might help avoid confusion when different servers have different values due to partition or restarts..